src/dispatch/static/dispatch/src/incident/ReportReceiptResources.vue

```diff
@@ -0,0 +1,218 @@
+<template>
+  <v-container>
+    <v-row dense>
+      <v-col cols="12">
+        <v-card outlined elevation="0">
+          <v-list-item two-line>
+            <v-list-item-content>
+              <v-list-item-title class="text-h5"> Incident Details </v-list-item-title>
+              <v-list-item-subtitle>{{ reported_at | formatRelativeDate }}</v-list-item-subtitle>
```

Comment:
Isn't this time always going to render as `Today`? Should we remove it if so?

Reply:
Good point, will remove.